### PR TITLE
Adding two incidents in Orlando

### PR DESCRIPTION
**Pull request by Ghostwaffles**
Couldn't find date for first incident, not entirely sure if second one belongs.

### CODE REVIEW

File: reports/Florida.md [hunk 1]
```diff
@@ -37,3 +38,20 @@ A police officer grabs a protestor from behind (light grey hoodie), pull him out
...
+* https://twitter.com/greg_doucette/status/1269017349727928320
+
+
+### Police box in protesters before curfew for mass arrest | June 3rd
+
+Police form barricades using bicycles to trap protesters leaving before curfew. 
+
+**Links**
+
+* https://twitter.com/filfranca/status/1268343603832774659
```
Comment (maxspencer) [inline reports/Florida.md:57]: I don't think this video is evidence of police brutality, we can't really see where the protestors are being directed, the video cuts out too soon.

File: reports/Florida.md [hunk 2]
```diff
@@ -37,3 +38,20 @@ A police officer grabs a protestor from behind (light grey hoodie), pull him out
 **Links**
 
 * https://twitter.com/ewcfilms/status/1268003421786906627
+
+### Police fire on protesters outside city hall | Date unknown
+
+Police open fire on protesters outside of city hall with teargas, flashbands, and apparently rubber bullets.
```
Comment (maxspencer) [inline reports/Florida.md:44]: Could you get a confirmation (e.g. using Google Streetview) of the `city hall` location claim?

Comment (Ghostwaffles) [reply reports/Florida.md:44]: Got it
Street is 29 W. South Street
I put 4 comparison shots into this album: https://imgur.com/a/HQC9sZk

Set 1: Video begins at intersection with S. Orange Ave facing east. You can see the concrete divider in the middle of the road and the turn arrow on the left. The steeple is visible in both backgrounds.

Set 2: Video turns towards buildings south of the street. Same window and pillar design. 

Set 3. Near intersection of  29 W. South Street and Boone Ave, facing east. Building shown in both are the same, and you can see the maintenance (?) doors on the left of both pictures. 

Set 4. Facing south on Boone Ave. Video image is blurry, but crosswalk and overpass are both visible.

Comment (maxspencer) [reply reports/Florida.md:44]: Fantastic work @Ghostwaffles, I'll add that detail and then merge this one 👍

**Pull request merged**